README.md

```diff
@@ -66,7 +66,15 @@ yarn fix
 ```
 yarn test
 ```
+### Setting up
```

Comment:
This is a bit confusing because the rest of this file is geared towards developers working on this project. If we want to add usage documentation, I think we should either create a new, separate section of the readme or create a new file. This information would ideally be on a real site - I wonder if we could add documentation on the Prettier website once this becomes stable?

Reply:
Actually I think adding documentation should be considered and well also contributor guide in the repositories on how to go about Contributing to Prettier. I don't think I have been it anywhere

Reply:
By creating another section, do I use wiki pages? for that or I should just leave it for it to be included in the docs?

Reply:
That's a really great point about missing a `CONTRIBUTING.md` document. We should definitely add this!

I think a Wiki Page would be great! @lipis @nickmccurdy What are your thoughts?

Reply:
The standard on GitHub is putting development information in CONTRIBUTING.md, it will even notify users about the file if they open a pull request for the first time. It seems like our development and help sections should be moved to contributing, while installation and this new setting up section should be in the readme. While we're at it, it would be nice to show some basic build instructions for Firefox users until we have it officially published.